controllers/generateConfigController.js

```diff
@@ -0,0 +1,190 @@
+const fs = require("fs");
+const { validEngines } = require("../utils/translationEnginesUtils");
+const { confirmUserAction, promptUserInput } = require("../utils/promptUtils");
+const { parsePath } = require("../utils/getConfigPath");
+const { listFilesOnDirectory } = require("../utils/listFiles");
+const { supportedLanguagesCodes } = require("../utils/supportedLanguagesUtils");
+
+const _promptTranslationEngines = () => {
+  const translationEngines = [...validEngines];
+  const translationEnginesToUse = [];
+
+  console.log(
+    "Will ask you for the translation engines you want to use. You will be able to change them later in the configuration file."
+  );
+
+  for (translationEngine of translationEngines) {
+    const shouldUseEngine = confirmUserAction(
+      `Do you want to use ${translationEngine} as translation engine? (y/n): `
+    );
+
+    if (shouldUseEngine) {
+      translationEnginesToUse.push(translationEngine);
+    }
+  }
+
+  return translationEnginesToUse;
+};
+
+const _promptBasePath = async () => {
+  let confirmedAction = false;
+  let pathFiles = [];
+  let basePath = "";
+
+  do {
+    let hasError = false;
+    basePath = promptUserInput(
+      'Base path for the translations files: b.e "src/localizations.": '
```

Comment:
What is `b.e`? Maybe a typo?

Reply:
by example, not sure if it is actually used heh, I also know e.g, what do you think is more common? or better not use acronyms?

Reply:
I remember e.g. more, I think is more "scholarly" but used in computer contexts, too

Reply:
It seems like the other option is `i.e.` 

Ref: https://grammarist.com/usage/ie-eg/

What I understand is

- i.e. similar to "in other words", you try to say the same simpler.
- e.g. show an individual example of a case of the statement you are saying
- b.e. more rare, can be used to show more examples in the plural. I didn't find this used as an abbreviation.

Reply:
Thanks for the information!, will change it